frontend/src/App.tsx

```diff
@@ -36,6 +36,7 @@ const queryClient = new QueryClient({
     queries: {
       staleTime: 3 * 60_000,
       refetchOnWindowFocus: false,
+      retry: false,
```

Comment:
Will this affect the rest of the application? AFAIU, if something fails, it will not be attempted to be fetched again. Won't this turn off the "live" UI when there's an error?
Maybe this is actually desirable in cases where things fail, but we need to have a way for the users to retry without having to actually refresh the application.

Reply:
by default react-query will retry 3 times and then gives up and returns an error. 
I disabled retries here so that the fallback is executed faster (without having to wait for 3 retries)
But this might not be a good idea, I'll remove this commit